<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArdaEditorForm</class>
 <widget class="QWidget" name="ArdaEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>536</width>
     <height>629</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="btnSourceText">
      <property name="text">
       <string>Исходное состояние текста</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="btnReplace">
      <property name="text">
       <string>Заменить</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Вернуть текст с шагом назад</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Размер текста</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Найти...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>511</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
